lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_download_file_by_version.rb

```diff
@@ -57,6 +60,12 @@ def self.available_options
                                        description: 'The prefix which is used in the GitHub release title',
                                        type: String,
                                        optional: true),
+          FastlaneCore::ConfigItem.new(key: :github_token,
+                                       env_name: 'GITHUB_TOKEN',
+                                       description: 'The GitHub OAuth access token',
+                                       optional: false,
+                                       default_value: false,
+                                       type: String),
```

Comment:
Self-Review: As stated on the P2, I'm not so sure if I should use the `GITHUB_TOKEN` or the `GHHELPER_ACCESS` as the env_name of the config Item.